apps/examples/lwnl_sample/lwnl_sample.c

```diff
@@ -75,7 +75,7 @@ static void *event_listener(void *arg)
 			}
 			memcpy(&status, buf, sizeof(lwnl_cb_status));
 			memcpy(&len, buf + sizeof(lwnl_cb_status), sizeof(uint32_t));
-			printf("[%d] status (%d) len(%d)\n", idx, status, len);
+			printf("[%d] status (%d) (%d) len(%d)\n", idx, status.type, status.evt, len);
```

Comment:
How about adding descriptions to log message like `status type (%d) evt (%d)` for easy debugging?

Reply:
I think printing raw value is enough to debug.